utils/cligen/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2020-2023 Olof Hagsand and Rubicon Communications, LLC(Netgate)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cligen
+PKG_VERSION:=6.4.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Olof Hagsand <olof@hagsand.se>, Philip Prindeville <philipp@redfish-solutions.com>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/clicon/$(PKG_NAME)/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=d12c43c2e1ac2c4c564b2e41cdc9902b540dd18a263af1dc4a6ed47cd08e102c
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/cligen
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=https://www.cligen.se
+  TITLE:=CLIgen is a Command-Line Interface generator
+  DEPENDS:=libxml2
+endef
+
+define Package/cligen/description
+  CLIgen is a Command-Line Interface generator
+endef
+
+CONFIGURE_ARGS+= \
+	--exec-prefix=/usr
+
+CONFIGURE_ARGS:=$(filter-out --disable-dependency-tracking,$(CONFIGURE_ARGS))
+
+INSTALLFLAGS:=-s
+ifneq ($(STRIP),:)
+INSTALLFLAGS+= \
+	--strip-program="$(STRIP)"
+endif
+
+CONFIGURE_VARS+= \
+	INSTALLFLAGS='$(INSTALLFLAGS)'
+
+$(info STRIP="$(STRIP)")
+$(info CONFIGURE_VARS="$(CONFIGURE_VARS)")
+$(info INSTALLFLAGS="$(INSTALLFLAGS)")
+
+define Build/InstallDev
+	echo "STRIP=$$(STRIP)"
+	echo "INSTALLFLAGS=$$(INSTALLFLAGS)"
+	$(INSTALL_DIR) $(1)/usr/include
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/cligen $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/lib
+	file $(PKG_INSTALL_DIR)/usr/lib/libcligen.so*
```

Comment:
replace to ```$(STRIP) $(PKG_INSTALL_DIR)/usr/lib/libcligen.so*```

Reply:
The strip should actually be happening inside the package's own `Makefile`, not here.